GsaGH/Parameters/1_Properties/GsaProp2d.cs

```diff
@@ -4,289 +4,235 @@
 using System.Globalization;
 using System.Linq;
 using GsaAPI;
+using GsaGH.Helpers.GsaAPI;
 using OasysUnits;
 using OasysUnits.Units;
-using OasysGH;
-using GsaGH.Helpers.GsaAPI;
-using System.Collections.ObjectModel;
-using Grasshopper.Kernel.Types;
 using Rhino.Geometry;
 
-namespace GsaGH.Parameters
-{
+namespace GsaGH.Parameters {
   /// <summary>
   /// Prop2d class, this class defines the basic properties and methods for any <see cref="GsaAPI.Prop2D"/>
   /// </summary>
-  public class GsaProp2d
-  {
+  public class GsaProp2d {
     #region fields
-    private int _id = 0;
+    private int _id;
     private Guid _guid = Guid.NewGuid();
```

Comment:
should not the public stuff be on top?

Reply:
yup should be, but for this task I'm doing only renaming